### ISSUE TITLE
[Issue?] picotts uses only female voices

### ISSUE THREAD

**Issue by Olf0** [rinigus/pure-maps#251]
picotts uses only female voices, regardless of the language used, both when navigating and when using the test function.
I believe (!) to have tested picotts successfully in Pure / Whogo / Poor Maps with a male voice a long time (many months) ago, although I usually have a female voice selected.  I do remember that not all languages I tried support both female and male voices, but IIRC some did.

If my memory is just wishful thinking, i.e. incorrect and picotts solely supports female voices, then please close this.

**Comment by rinigus** [comment]
Nope, it only has female voices. Closing

**Comment by Pohli** [comment]
@Olf0: Maybe you mix it up with Mimic. I tried out Mimic many months ago and I think to remember that I could successfully switch between male and female voices.

**Comment by Olf0** [comment]
@Pohli, that is very unlikely, because I had mimic installed only for a few days about 1,5 years ago in my Jolla 1s, because of @rinigus' general recommendation for it.  But it didn't work (garbled output at the command line, no voice output in Poor Maps), so I switched to picotts, which was working out of the box (at the command line and basically working in Poor Maps / Whogo Maps).
But one's memory is sometimes astonishingly selective, so I cannot really exclude that explanation.  Or it reminded me of some similar language selector in one of the other Map / Navigation apps I tested lately.

Never mind.
